.changeset/strange-squids-turn.md

```diff
@@ -0,0 +1,44 @@
+---
+'@openzeppelin/defender-sdk-example-create-action': patch
```

Comment:
Do we still have to submit this even for snapshot release? Thinking if the changeset will create PR with version packages since we are committing this file